src/main/kotlin/baseball/domain/ComputerBall.kt

```diff
@@ -0,0 +1,14 @@
+package baseball.domain
+
+private const val START_INDEX = 0
+
+class ComputerBall(val numbers: List<Int>) {
+
+    fun countContainingBalls(balls: List<Int>): Int {
+        return numbers.count { number -> balls.any { ball -> number == ball } }
+    }
+
+    fun countCorrectBalls(balls: List<Int>): Int {
+        return (START_INDEX until numbers.size).count { idx -> numbers[idx] == balls[idx] }
```

Comment:
👍
infix 함수 좋죠..ㅎㅎ

Reply:
Collection에서 index를 사용하고 싶을 때, `filteredIndex`를 활용할 수도 있어요!
```suggestion
        return numbers.filterIndexed { index, number -> number == balls[index] }.size
```

더 줄이면 아래처럼!

```kotlin
    fun countCorrectBalls(balls: List<Int>) = 
        numbers.filterIndexed { index, number -> number == balls[index] }.size
```

Reply:
@TaeyeonRoyce  코틀린 고수시네요

Reply:
```java
return numbers.zip(balls).count { (a, b) -> a == b }
```

전 이렇게 했어요 zip하면 같은 인덱스끼리 묶어준다고........

Reply:
와 이런 건 진짜 신박하네요..ㄷㄷ